rules/docker-router-advertiser.mk

```diff
@@ -0,0 +1,12 @@
+# Docker image for router advertiser
+
+DOCKER_ROUTER_ADVERTISER = docker-router-advertiser.gz
+$(DOCKER_ROUTER_ADVERTISER)_PATH = $(DOCKERS_PATH)/docker-router-advertiser
+$(DOCKER_ROUTER_ADVERTISER)_LOAD_DOCKERS = $(DOCKER_CONFIG_ENGINE)
+SONIC_DOCKER_IMAGES += $(DOCKER_ROUTER_ADVERTISER)
+SONIC_INSTALL_DOCKER_IMAGES += $(DOCKER_ROUTER_ADVERTISER)
+
+
+$(DOCKER_ROUTER_ADVERTISER)_CONTAINER_NAME = router_advertiser
```

Comment:
how about radvd?

Reply:
As long as you think it's clear enough, I can abbreviate it. I was just concerned about clarity.

Reply:
Opted for "radv" (**r**outer **adv**ertiser) so that the docker name isn't tied to the specific radvd daemon, in case we ever decide to change to a different solution in the future.